tests/garage/envs/test_sliding_mem_env.py

```diff
@@ -0,0 +1,19 @@
+import pickle
+import unittest
+
+from garage.envs.box2d import CartpoleEnv
+from garage.envs.sliding_mem_env import SlidingMemEnv
+from tests.helpers import step_env
+
+
+class TestSlidingMemEnv(unittest.TestCase):
+    def test_pickleable(self):
+        inner_env = CartpoleEnv(obs_noise=5.)
+        env = SlidingMemEnv(inner_env, n_steps=10)
+        round_trip = pickle.loads(pickle.dumps(env))
+        assert round_trip
+        assert round_trip.n_steps == env.n_steps
+        assert round_trip.env.obs_noise == env.env.obs_noise
+        step_env(round_trip)
+
+    test_pickleable.broken = True
```

Comment:
Is there an issue filed to fix this?

Reply:
All of these have the broken flag:
```
tests/garage/envs/mujoco/gather/test_swimmer_gather_env.py:16:    test_pickleable.broken = True
tests/garage/envs/mujoco/gather/test_point_gather_env.py:16:    test_pickleable.broken = True
tests/garage/envs/mujoco/gather/test_ant_gather_env.py:16:    test_pickleable.broken = True
tests/garage/envs/mujoco/maze/test_swimmer_maze_env.py:16:    test_pickleable.broken = True
tests/garage/envs/mujoco/maze/test_point_maze_env.py:16:    test_pickleable.broken = True
tests/garage/envs/mujoco/maze/test_ant_maze_env.py:16:    test_pickleable.broken = True
tests/garage/envs/test_sliding_mem_env.py:19:    test_pickleable.broken = True
tests/garage/envs/test_occlusion_env.py:20:    test_pickleable.broken = True
```
But the flag is not used with any nose2 configuration (I checked with grep), and if I run:
```
nose2 -c setup.cfg -A broken
```
All of the tests are passing.

I will remove this flag since it's not used anywhere else and the tests are passing. There's an issue for the rendering of the GatherEnv but it's already documented at #323.